### DISCUSSION TITLE
Daily 11/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#299] [General]
Ayer edité y subí el video a la Wiki además de poner vínculos de backup en Dropbox y YouTube. También moví los proyectos a los directorios finales. También actualicé las instrucciones con los nuevos directorios en el README, tengo que actualizar las instrucciones en el directorio de documentación. El sitio se deploya correctamente en el nuevo directorio. Hoy voy a subir la versión de AWS y linkear el front-end al AWS. Hoy habría que terminar con el sitio de Wordpress también. Y si quedan conformes con los diagramas de caso de uso mergearíamos a main y ya no tocaríamos más el TP.

**Comment by NataliaAlvarezIspc** [comment]
Ayer probe el sitio deployado en github, estaría funcionando bien. También hice una página de wordpress para el portfolio del grupo. Hice el diagrama de casos de uso general. Por la tarde y noche participé de las clases sincrónicas de SGC y Full Stack. No pude hacer las pruebas unitarias.
Hoy tuve clase sincrónica de Inglés. Voy a estar revisando el diagrama de casos de usos, cuando me den el ok lo subo al repo. Trataré de hacer las pruebas unitarias.

**Comment by AhumadaCeleste** [comment]
Ayer pude completar la última lección de ingles. También avance un poco con el modulo final de Python. Voy a avanzar con el estudio de JavaScript pero no voy a subir nada por temor a dañar lo echo a tan poco de entregar. También veré las clases de ayer. Este fin de semana quisiera terminar con Python. Me queda mucho de Backend de contenido que no alcance a estudiar. Me gustaría poder incorporarlo antes de que retiren el contenido de la pagina.
